plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergRestCatalogConnectorSmokeTest.java

```diff
@@ -116,6 +119,18 @@ protected String getMetadataLocation(String tableName)
         throw new UnsupportedOperationException("metadata location for register_table is not supported");
     }
 
+    @Override
+    protected String generateTableLocation(String tableName)
+    {
+        return String.format("%s/%s/%s", warehouseLocation, getSession().getSchema(), tableName);
```

Comment:
nit: Static import `format` method.

Reply:
Reminder.

Reply:
Sorry, I missed it. Thanks for the reminder. Updated now.